requirements/lite/base.txt

```diff
@@ -3,7 +3,7 @@
 
 numpy>=1.17.2, <1.23.1
 torch>=1.9.*, <1.13.0
-fsspec[http]>=2021.06.1, <2022.6.0
+fsspec[http]>2021.06.0, <2022.6.0
```

Comment:
this is a hack to the system as for oldest we replace `>=` by `==` and the the fixed version from PL and App was not compatible so I made this lower and `>` only but the meaning/range is identical